### ISSUE TITLE
Mailing list link dead

### ISSUE THREAD

**Issue by Flohack74** [ubports/ubports-app#56]
See https://github.com/ubports/ubports-app/blob/e6262a5589bbb42259d60eb4943aead1e1d58316/po/de.po#L211 - The link does not work at all. Should this go here? https://list.ubports.com/mailman/listinfo

**Comment by NeoTheThird** [comment]
Yep, it should. These ubports.com/r/app* links were requested by the website team and I always had a bad feeling about doing it this way. Now they abolished them without telling me. Great. Communitcation!
